homeassistant/components/point/__init__.py

```diff
@@ -143,7 +143,9 @@ async def async_setup_entry(hass: HomeAssistant, entry: PointConfigEntry) -> boo
     entry.runtime_data = PointData(client)
 
     await async_setup_webhook(hass, entry, point_session)
-    # Entries are added in the client.update() function.
+    await hass.config_entries.async_forward_entry_setups(
+        entry, [*PLATFORMS, Platform.ALARM_CONTROL_PANEL]
+    )
```

Comment:
Why isn't the alarm paltform added to the PLATFORMS list?

Reply:
Because it is handled differently here: https://github.com/home-assistant/core/pull/126535/files#diff-8460d726be26ad1ac12e19955747c29ed546fb08f2ce78566f034d9429abecc7R248